src/cli.js

```diff
@@ -178,28 +189,58 @@ async function main() {
     process.exit(1);
   }
 
+  if (projectRootDirectory) {
+    try {
+      const stat = await fs.stat(projectRootDirectory);
+      if (!stat.isDirectory()) {
+        console.error(
+          `Project root must be a directory: '${projectRootDirectory}'`,
+        );
+        process.exit(1);
```

Comment:
This is probably out of scope for this PR, but I recently realized how strongly Node advises against the log-and-process.exit pattern: https://nodejs.org/api/process.html#process_process_exit_code

Going through `snaps-cli` and getting rid of 17 cases of the same is on my TODO-list 😑

Reply:
TIL. Easy enough to avoid this pattern in this case at least. I can fix this in a follow-up PR.

Reply:
Resolved here: https://github.com/MetaMask/auto-changelog/pull/43